.opensource/project.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
What is this file for?

Reply:
It controls the appearance of this page
https://firebaseopensource.com/projects/firebase/firebase-js-sdk/

Reply:
Pretty cool!